### PR TITLE
Adding the international shipping notice

### LINKED ISSUE #187: Add international shipping message to Shopify cart.
Let's add a warning about international shipping to the Shopify cart so we're setting the right expectation. It should say something like:

'International Shipping is unpredictable and slow! Please be advised that the shipment might take much longer than expected.'

### PR DESCRIPTION
**Pull request by AnushPatel**
Can someone just give me pointers on how to test this on my local end? The syntax seems right in my opinion but it probably won't render as I want..

closes #187

### CODE REVIEW

**Comment by jlengstorf** [comment]
@AnushPatel if you clone this locally and run `gatsby develop`, does the site start properly? There shouldn't be any special steps required to get things to build, but please let me know if that's not the case.

Looking at this, I suspect it's going to be too much text for the space it's been put in, but I haven't verified yet.

Can you give the local dev setup a shot and see if that works for you?

Thanks!

**Comment by AnushPatel** [comment]
@jlengstorf Why didn't that idea come to me earlier! Let me give that a shot

**Comment by AnushPatel** [comment]
@jlengstorf The storefront loads a OK - Its at checkout which is where I can't access the local copy. I had a better idea than this PR but I will have to test it.

**Comment by AnushPatel** [comment]
I am thinking of putting the Information in the 'About the Gatsby store' section along with 'Got any more questions?' but I'm more of a VUI Guy than this so what do you think @jlengstorf?

**Comment by jlengstorf** [comment]
@AnushPatel I'm not sure I understand what you mean. You want to put the notice in the footer? I think that's probably fine for now, and will likely be easier to style than a big block of text in the cart.

Want to give it a shot here?

Thanks!

**Comment by AnushPatel** [comment]
@jlengstorf Sorry! Explained that badly, Yes, I would like to put the message on the footer for the time being for the same reason as you outlined (Its quite dificult to style the text on the cart) and for the fact that the design is getting changed anyways so it can be implimented then into other places such as the cart.

I will commit to the PR the changes sometime today.


Thanks for the help!

**Comment by jlengstorf** [comment]
I saw your PR in #189, so I'm going to close this.

**Comment by jlengstorf** [comment]
I just realized you did this PR into your own branch. Sorry about that! Let me reopen.

File: src/components/Cart/OpenCart.js [hunk 1]
```diff
@@ -146,6 +146,8 @@ class OpenCartComp extends React.Component {
                       </CostDetails>
                       <CostDetails>
                         Shipping: <PriceBox>FREE</PriceBox>
+                        International shipments can take six weeks or more to be delivered. 😱 Tracking updates may not always show up in real time on your tracking link. If you still have not received your order at the end of six weeks, please let us know by sending an email to team@gatsbyjs.com, and we will see how we can help
```
Comment (jlengstorf) [inline src/components/Cart/OpenCart.js]: Should we remove this now that you've added it to the footer?

Comment (AnushPatel) [reply src/components/Cart/OpenCart.js]: Yes Please! How do I completely remove that commit (Sorry - Haven't really done these types of PR)

Comment (jlengstorf) [reply src/components/Cart/OpenCart.js]: You can edit the file here to remove the changes. It'll be a new commit, but that's not a problem.

Thanks!

Comment (jlengstorf) [reply src/components/Cart/OpenCart.js]: Oh, you already got this figured out. 😅

Comment (AnushPatel) [reply src/components/Cart/OpenCart.js]: Yeah 🙂 Thanks for the help though 👍

**Comment by jlengstorf** [comment]
Here's how it looks. @lindaleebumblebee any feedback before we merge?

screen shot 2018-11-29 at 10 01 51 am

**Comment by AnushPatel** [comment]
NGL -  Does look better than I was expecting but lets see what @lindaleebumblebee thinks..

**Comment by greglobinski** [comment]
I think that after merging https://github.com/gatsbyjs/store.gatsbyjs.org/pull/201 the PR will be outdated.

screenshot from 2019-01-02 11-12-34

screenshot from 2019-01-02 11-12-45

**Comment by AnushPatel** [comment]
@greglobinski That is correct! I will close this PR. Thanks for giving me the heads up.

**Pull request closed without merging**